hugegraph-server/hugegraph-rocksdb/src/main/java/org/apache/hugegraph/backend/store/rocksdb/RocksDBStore.java

```diff
@@ -423,6 +423,16 @@ public void close() {
         this.closeSessions();
     }
 
+    public void forceClose() {
+        try {
+            this.checkOpened();
+            this.closeSessions();
+        } catch (Throwable ignore) {
+            return;
+        }
+        ((RocksDBStdSessions)this.sessions).forceClose();
```

Comment:
ok.
but we still need to iterate this.sessions() and close every session.

Reply:
yes, now, this way close the session first, last forceClose